File: src/main/java/me/mixces/ornithe_animations/OrnitheAnimations.java
package me.mixces.ornithe_animations;

import org.apache.logging.log4j.LogManager;
import org.apache.logging.log4j.Logger;

import net.ornithemc.osl.entrypoints.api.ModInitializer;

public class OrnitheAnimations implements ModInitializer {

	public static final Logger LOGGER = LogManager.getLogger("Ornithe Animations");

	@Override
	public void init() {
		LOGGER.info("initializing example mod!");
	}
}

File: src/main/kotlin/me/mixces/ornitheanimations/OrnitheAnimations.kt
package me.mixces.ornitheanimations

import me.mixces.ornitheanimations.config.Config
import net.ornithemc.osl.config.api.ConfigManager
import net.ornithemc.osl.entrypoints.api.ModInitializer

object OrnitheAnimations : ModInitializer {

    override fun init() {
        ConfigManager.register(Config)
    }
}
